draft-ietf-rats-eat.md

```diff
@@ -1078,17 +1078,17 @@ The values for the results enumerated type are as follows:
 ~~~~
 
 
-### Submodules (submods) {#submods}
+### submods (Submodules) Claim {#submods}
```

Comment:
Would like to leave this change out of this PR and handle it in the submodules rework PR.

Reply:
I dropped the word "claim" from the title here, restored the use of the word "section" in the IANA section and restored the old prose for this section, but left the change to have JWT name (Claim name) in the title.